Forta-Bot-Challenge-4/COMP01/README.md

```diff
@@ -0,0 +1,29 @@
+# Investment tracker bot for USDC
+
+## Description
+
+The COMP01 bot is a monitoring tool designed to track individual user investments over a 24-hour period, generating alerts when the user surpasses a predefined threshold during supply events. The associated proxy contract for this bot is [`0x1B0e765F6224C21223AeA2af16c1C46E38885a40`](https://etherscan.io/address/0x1B0e765F6224C21223AeA2af16c1C46E38885a40).
```

Comment:
You use the term "predefined", but from the addition of the function calculating the average, it is no longer "predefined". So this should be updated.